doc/source/indexing.rst

```diff
@@ -248,10 +249,10 @@ new column. In 0.21.0 and later, this will raise a ``UserWarning``:
 
 .. code-block:: ipython
 
-    In[1]: df = pd.DataFrame({'one': [1., 2., 3.]})
-    In[2]: df.two = [4, 5, 6]
+    In [1]: df = pd.DataFrame({'one': [1., 2., 3.]})
+    In [2]: df.two = [4, 5, 6]
     UserWarning: Pandas doesn't allow Series to be assigned into nonexistent columns - see https://pandas.pydata.org/pandas-docs/stable/indexing.html#attribute_access
-    In[3]: df
+    In [3]: df
     Out[3]:
```

Comment:
does the `Out` need a space before the `[` too?

Reply:
Nope, If the `:` align vertical it's all good. I think it's done for visual beauty.